bpm/bonita-synchro-repository/bonita-synchro-register/src/main/java/org/bonitasoft/engine/synchro/JMSProducer.java

```diff
@@ -33,8 +32,12 @@
 
     private static JMSProducer jmsProducer;
 
-    private JMSProducer(final long timeout) throws JMSException {
-        final String brokerURL = System.getProperty("broker.url");
+    private JMSProducer(final long timeout, String brokerURL) throws JMSException {
+
+        // if brokerURL not defined before, get property
+        if (brokerURL == null) {
+            brokerURL = System.getProperty("broker.url");
+        }
```

Comment:
it was for upward compatibility 
engine can work with old versions of performance tool with this